### PR TITLE
filters and input on body of page

### PR DESCRIPTION
**Pull request by SenadK1997**

### CODE REVIEW

**Comment by SenadK1997** [comment]
Need a little help for styling option box. Couldn't find in the root of inspect element anything that can change style of the box.

File: src/components/BodyPage.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        data() {
+            return {
+                selected: 'Test'
+            }
+        }
+    }
+</script>
+    
+<template>
+    <section class="c-section__filters">
```
Comment (kenanbalija) [inline src/components/BodyPage.vue:61]: This does not have to be **c-section__filters**.
It's basically a new component which you could call **c-filters**

File: src/components/BodyPage.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <div class="c-section__filters__parent">
+            <div class="c-section__filters-input">
```
Comment (kenanbalija) [inline src/components/BodyPage.vue:63]: This would be like **c-filters__input**

File: src/components/BodyPage.vue [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                <svg stroke="currentColor" fill="none" stroke-width="2" viewBox="0 0 24 24" stroke-linecap="round" stroke-linejoin="round" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg"><circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line></svg>
+                <input type="text" id="search-input" placeholder="Search for a country...">
+            </div>
+            <div class="c-section__filters-select__parent">
```
Comment (kenanbalija) [inline src/components/BodyPage.vue:67]: **c-filters__select**

File: src/components/BodyPage.vue [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                <select placeholder="Filter by Region" class="c-section__filters-select__child">
```
Comment (kenanbalija) [inline src/components/BodyPage.vue]: **c-filters__select-input**

File: src/components/NavBar.vue [hunk 5]
```diff
@@ -11,24 +11,30 @@ export default {
 
 <template>
     <section class="c-section">
```
Comment (kenanbalija) [inline src/components/NavBar.vue:13]: You should call this: **c-navbar**

File: src/components/NavBar.vue [hunk 6]
```diff
@@ -11,24 +11,30 @@ export default {
...
-        <div class="c-section__header"><h1>Where in the world?</h1></div>
-        <div class="c-section__mode">
-            <svg stroke="currentColor" fill="none" stroke-width="0" viewBox="0 0 24 24" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"></path></svg>
-            <button>Dark Mode</button>
+        <div class="c-section__parent__div">
```
Comment (kenanbalija) [inline src/components/NavBar.vue:14]: & then this would be **c-navbar__container**

File: src/components/NavBar.vue [hunk 7]
```diff
@@ -11,24 +11,30 @@ export default {
...
+            <div class="c-section__header"><h1>Where in the world?</h1></div>
```
Comment (kenanbalija) [inline src/components/NavBar.vue:15]: **c-navbar__container-header**

File: src/components/NavBar.vue [hunk 8]
```diff
@@ -11,24 +11,30 @@ export default {
...
+            <div class="c-section__mode">
```
Comment (kenanbalija) [inline src/components/NavBar.vue:16]: **c-navbar__container-button**

File: src/main.js [hunk 9]
```diff
@@ -1,5 +1,26 @@
...
+import {createRouter, createWebHistory} from 'vue-router';
+import Countries from '../views/Countries.vue'
+import DetailsPage from '../views/DetailsPage.vue'
 
-createApp(App).mount('#app')
+const routes = [
+    {
+        path: '/',
+        component: Countries,
+        name: 'Countries'
```
Comment (kenanbalija) [inline src/main.js:12]: Name should be lowercase, so change this from `name: 'Countries'` to `name: 'countries'`

File: src/main.js [hunk 10]
```diff
@@ -1,5 +1,26 @@
...
+    },
+    {
+        path: '/details',
+        component: DetailsPage,
+        name: 'DetailsPage'
```
Comment (kenanbalija) [inline src/main.js:17]: same as before but also change casing from `name: 'DetailsPage'` to `name: 'details-page'`

**Review by kenanbalija — COMMENTED** [review]
Check the comments.

**Pull request merged**